vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -50,7 +50,7 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
-      new_result.merge(
+      new_result.merge!(
```

Comment:
Looks like this can all be reduced to:

```ruby
parse_base_item(service).merge(
  :port   => service.spec.port,
  ... etc.
)
```

I don't know if @Fryguy approves of that style, though :smile: